test/integration/targets/service_facts/tasks/main.yml

```diff
@@ -3,10 +3,6 @@
 # Copyright: (c) 2020, Abhijeet Kasurde <akasurde@redhat.com>
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
-- name: skip broken distros
-  meta: end_host
-  when: ansible_distribution == 'Alpine'
-
```

Comment:
This was removed in https://github.com/ansible/ansible/pull/84379/files#diff-47b9b80830e40ec6852ad6845455417159a8d06a18b4856172e94b26590b78c8L6-L9.